cli/gumball-machine/templates/initializeGumballMachine.ts

```diff
@@ -0,0 +1,54 @@
+import log from 'loglevel';
+import { BN, Provider, Program } from "@project-serum/anchor";
+import {
+  PublicKey,
+  Keypair,
+  SystemProgram,
+  Transaction,
+  Connection as web3Connection,
+  LAMPORTS_PER_SOL,
+} from "@solana/web3.js";
+import {
+    gumballMachineHeaderBeet,
+    InitializeGumballMachineInstructionArgs,
+    EncodeMethod
+} from "../../../contracts/sdk/gumball-machine";
+import { NATIVE_MINT } from "@solana/spl-token";
+import {
+  getMerkleRollAccountSize,
+} from "../../../contracts/sdk/gummyroll";
+import {
+    val,
+    strToByteArray,
+    strToByteUint8Array
+} from "../../../contracts/sdk/utils/index";
+
+export const gumballMachineInitArgs: InitializeGumballMachineInstructionArgs = {
+  maxDepth: 3,
+  maxBufferSize: 8,
+  urlBase: strToByteArray("https://arweave.net", 64),
+  nameBase: strToByteArray("GUMBALL", 32),
+  symbol: strToByteArray("GUMBALL", 8),
```

Comment:
Do these arguments get automatically set if I fail to set these manually from the CLI?

Reply:
yeah, the idea behind these files is that many of the instructions, such as this one, have too many args to specify each at the CLI. So the user of the CLI would just input the args into these files directly.